.github/workflows/CI.yml

```diff
@@ -134,6 +134,10 @@ jobs:
         os: [ macos-12, ubuntu-20.04, windows-2019 ]
         node: [ '16', '18' ]
         vscode: [ 'stable' ]
+        # latest version of daffodil doesn't seem to work on windows-2019 running node 18
+        exclude:
+          - os: windows-2019
+            node: 18
```

Comment:
Is there any information why tests fail with this specific combination? Feels like this might bug in Daffodil 3.7.0 that I don't want to just ignore.

Reply:
I actually probably shouldn't say its daffodil as I am not sure why causes it. The main issues that get thrown are that commands that are being registered aren't available. I am not sure why daffodil would be causing it, not sure if its just something broken on node js 18 for windows.

Reply:
It looks like a bunch of nightlies are failing too. Not sure if those are expected or if that's a sign that the same issue happens with 3.6.0, and might support this not being a daffodil issue.

Reply:
Yeah I agree. Some of the nightly tests are outliers where they kind of fail randomly due to network issues such as not being able to install yarn packages or something. The main issue I am seeing is that commands are not becoming available for Windows when using NodeJS 18. I am not sure why, I will have to do testing in a VM to see if I can reproduce the issue. Hopefully this will help me find the issue and fix it

Reply:
@stevedlawrence I was able to find a fix. I booted up a windows VM and installed nodejs 18.20.2 (current latest of 18). When I did this I had the same issues as CI, but when I installed the next latest version available on windows using `choco`, this version being `18.18.2` everything ran fine. So I made the CI use nodejs 18.20.1 which seems to work fine so there must be a bug between windows and nodejs 18.20.2 that causes this issue.

I plan to do a little bit more digging to see if I can somehow get 18.20.2 to work but I think I should make that a separate PR in that case. This being since I may not be able to get it to work and may need to wait for 18.20.3 or 18.21.0

Reply:
A separate PR sounds good to me.